search-provider/eks-discovery-feed-provider-dbus.xml

```diff
@@ -39,4 +39,10 @@
       <arg type="aa{ss}" name="Results" direction="out" />
     </method>
   </interface>
+  <interface name="com.endlessm.DiscoveryFeedVideo">
+    <method name="GetRelevantVideo">
```

Comment:
Like in the app store case, we can drop `Relevant` from the API method name. Also, it should probably be `Videos`, since we return more than one.